src/controllers/organizationController.ts

```diff
@@ -127,4 +130,39 @@ export default class OrganizationController {
       );
     });
   }
+
+  public getDocument(
+    req: Request
+  ): Promise<
+    // tslint:disable-next-line:max-union-size
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseErrorNotFound
+    | IResponseErrorInternal
+    | IResponseSuccessPdf
+  > {
+    return withUserFromRequest(req, async user => {
+      if (user.role !== UserRoleEnum.ORG_DELEGATE) {
+        return ResponseErrorForbiddenNotAuthorized;
+      }
+      const filePath = `./documents/${req.params.ipaCode}/${req.params.fileName}`;
+      try {
+        await fs.promises.access(filePath);
+      } catch (error) {
+        return ResponseErrorNotFound(
+          "Not found",
+          "The requested document does not exist"
+        );
+      }
+      try {
+        const buffer = await fs.promises.readFile(
```

Comment:
consider using http://expressjs.com/en/api.html#res.download (or at least a read stream instead of loading the entire file into memory)